### PR TITLE
#602 Change post button to be more obvious

### LINKED ISSUE #602: Post Button
- [ ] Did you check to see if this issue already exists?
- [ ] Is this only a single feature request? Do not put multiple feature requests in one issue.
- [ ] Is this a question or discussion? Don't use this, use https://lemmy.ml/c/jerboa

** Describe the feature request below **

I'd like to see the Post Button, which is currently a "+” sign, be changed to the word Post, or some other blatantly obvious button, so the user doesn't think it's an " additional options" thing.  Even here on Github it's a blue arrow that stands out from the rest of the screen.

### PR DESCRIPTION
**Pull request by Chris-Kropp**
resolves #602 This changes the submit icon from `add` to `submit` and adds a preview for the post header.
Icon_only

### CODE REVIEW

**Review by twizmwazin — CHANGES_REQUESTED** [review]
Your change looks great. Can you fix the commit history?

**Comment by twizmwazin** [comment]
Run `./gradlew formatKotlin` to fix the formatting

File: app/src/main/java/com/jerboa/ui/components/post/create/CreatePost.kt [hunk 1]
```diff
@@ -74,11 +75,18 @@ fun CreatePostHeader(
                         color = MaterialTheme.colorScheme.onSurface,
                     )
                 } else {
-                    // Todo add are you sure cancel dialog
-                    Icon(
-                        Icons.Outlined.Add,
-                        contentDescription = stringResource(R.string.form_submit),
-                    )
+                    // Todo add are you sure cancel dialog+
+                    if (formValid && !loading) {
```
Comment (dessalines) [inline app/src/main/java/com/jerboa/ui/components/post/create/CreatePost.kt:79]: Loading was already checked above.

File: app/src/main/java/com/jerboa/ui/components/post/create/CreatePost.kt [hunk 2]
```diff
@@ -74,11 +75,18 @@ fun CreatePostHeader(
...
+                        Icon(
+                            Icons.Filled.Send,
```
Comment (dessalines) [inline app/src/main/java/com/jerboa/ui/components/post/create/CreatePost.kt:81]: Would be much easier to do `icon = if (formValid) { ...`

Comment (Chris-Kropp) [reply app/src/main/java/com/jerboa/ui/components/post/create/CreatePost.kt:81]: Yep, that's much cleaner

**Pull request merged**